lib/ui/src/components/sidebar/Tree.tsx

```diff
@@ -169,6 +170,11 @@ const Node = React.memo<NodeProps>(
               event.preventDefault();
               onSelectStoryId(item.id);
             }}
+            onMouseEnter={() => {
+              if (item.isLeaf) {
+                api.emit(PRELOAD_STORIES, [item.id]);
+              }
```

Comment:
We want to emit the signal when you mouse over the component *before* you click on the first story (there's no point in preloading sibling stories once they are already visible, right?).

I think something like this will do it (factoring in docs-only stories too)?

```suggestion
              if (item.isComponent) {
                api.emit(PRELOAD_STORIES, [item.isLeaf ? item.id : item.children[0]]);
              }
```